testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -171,6 +171,7 @@ def test_Universe_invalidpermissionfile_IE_msg(self):
         try:
             mda.Universe(os.path.join(temp_dir.name, 'permission.denied.tpr'))
         except IOError as e:
+            #pylint: ignore=unsupported-membership-test
```

Comment:
@kain88-de, `disable` vs. `ignore` certainly makes a difference for me under pylint 1.7.2 (the latter doesn't work), with the following example:
```python
# test.py
try:
    open('permission.denied.tpr')
except IOError as e:
    # pylint: disable=unsupported-membership-test
    assert 'Permission denied' in e.strerror
```
(also tested, with same result, using `numpy.testing.assert_`, which we decided should be replaced the plain `assert` anyway).

If I use `ignore` instead of `disable` I get
```test.py:5: [E1135(unsupported-membership-test), ] Value 'e.strerror' doesn't support membership test```
I'm using python 3.5. Maybe that's the reason?

Finally, the warning does make sense, in that `IOError` is initialized with `strerror` set to `None`. We could conceptually be catching such an `IOError` here, which would make the test error (instead of failing). Writing
```python
assert 'Permission denied' in str(e.strerror)
```
shuts up pylint and protects from such funky `IOError` scenarios.